requirements-dev.txt

```diff
@@ -5,7 +5,7 @@ python_dotenv==1.0.0
 pylint==2.17.4
 tox==4.6.4
 pytest-rerunfailures==12.0
```

Comment:
Can we drop this dep now we aren't doing global reruns, or do we have annotations somewhere?

Reply:
Yup I can remove it [d167f40](https://github.com/IBM/cloudant-python-sdk/pull/517/commits/d167f40f2c3cacc411b96dcee0e363576f920d29)